docs/meeting-notes/foam-core-2020-07-11.md

```diff
@@ -122,6 +122,6 @@ How others solve this:
 - Unique ids -- could support optionally as part of file name or front matter metadata. Should not be required.
 
 [//begin]: # "Autogenerated link references for markdown compatibility"
-[todo]: ../todo.md "Todo"
-[Index]: ../index.md "Foam"
+[todo]: ../todo "Todo"
+[Index]: ../../node_modules/socks/docs/examples/index "socks examples"
```

Comment:
interesting... I wonder where this is coming from - feels like a bug in the definition generation..

Reply:
Yeah, I will leave this alone for now.